src/components/JuzMarker/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { renderToString } from 'react-dom/server';
+import bindTooltip from 'utils/bindTooltip';
+import { intlShape, injectIntl } from 'react-intl';
+import { juzStart, juzAndHizbArabicNum } from './markNumbers';
+
+const juzMarkTop = require('../../../static/images/juzTopArt.svg');
+const juzMarkBottom = require('../../../static/images/juzBottomArt.svg');
+
+const StyledContainer = styled.div`
+  padding: 0.05em;
+  border: solid 1px #2CA4AB;
+  width: 0.659em;
+  margin-left: calc((100% - 0.659em) / 2);
+  margin-top: 0.055em;
+  margin-bottom: 0.055em;
+}
+`;
+
+const StyledJuzArt = styled.img`
+  height: 30px;
+  width: 0.659em;
+  margin-left: calc((100% - 0.659em) / 2);
+  display: block;
+`;
+
+const StyledJuzNumber = styled.div`
+  font-size: 10px;
+  width: 100%;
+  text-align: center;
+  display: block;
+`;
+
+const StyledAyah = styled.div`
+  margin-top: 3%;
+`;
+
+export const Decoration = ({ juzNumber }) => (
```

Comment:
Can we make this a separate component in a separate file?

Reply:
On it